gen/AArch64Compile_gen.ml

```diff
@@ -2764,7 +2764,14 @@ module Make(Cfg:Config) : XXXCompile_gen.S =
         | Data loc -> loc
         | Code lab -> match f with
           | CMO (_,Next) -> lab
-          | _ -> Warn.fatal "can't create address dependency for a code location without DC.CVAU or IC.IVAU" in
+          | _ -> Warn.fatal "Address dependency to code location must be directly followed by DC.CVAUn or IC.IVAUn" in
+        (try ignore (C.find_node
+          (fun m -> match m.C.edge.E.edge with
+            | E.Insert (CMO (_,Prev)) when m.C.evt.C.loc == n.C.evt.C.loc ->
```

Comment:
Are you sure about using physical equality?

Reply:
Hi @maranget no, thank you for spotting that. I have just changed it to use loc_compare instead.